dask_expr/_groupby.py

```diff
@@ -674,7 +676,7 @@ class GroupByReduction(Reduction, GroupByBase):
     _chunk_cls = GroupByChunk
 
     def _tune_down(self):
-        if len(self.by) > 1 and self.operand("split_out") is None:
+        if self.operand("split_out") is None:
```

Comment:
Wouldn't we always shuffle now?

Reply:
Yep

Reply:
except if split_out=1 is set explicitly

Reply:
We just had a conversation about this and agreed that we'll go for this automatic behavior. This means that some group by operations will perform a bit worse since we are forcing a shuffle that is not strictly necessary.
For large output results the shuffle is a necessity and for tiny output results, the additional shuffle step only adds a marginal performance penalty in our testing.

It is a safer choice and most users will not want to or be able to dig in deep enough to set this parameter such that this is a good default choice.